.eslintrc

```diff
@@ -0,0 +1,62 @@
+{
+  "root": true,
+  "env": {
+    "node": true,
+    "jest/globals": true
+  },
+  "parser": "@typescript-eslint/parser",
+  "parserOptions": {
+    "project": [
+      "tsconfig.json",
+      "tests/tsconfig.json",
+      "examples/tsconfig.json",
+      "benchmarks/tsconfig.json"
+    ],
+    "ecmaVersion": "latest",
+    "sourceType": "module"
+  },
+  "settings": {
+    "import/parsers": {
+      "@typescript-eslint/parser": [".ts"]
+    },
+    "import/resolver": {
+      "typescript": {
+        "alwaysTryTypes": true,
+        "project": [
+          "tsconfig.json",
+          "tests/tsconfig.json",
+          "examples/tsconfig.json",
+          "benchmarks/tsconfig.json"
+        ]
+      }
+    }
+  },
+  "plugins": ["import", "@typescript-eslint", "eslint-plugin-tsdoc", "prettier", "jest"],
+  "extends": [
+    "airbnb-base",
+    "airbnb-typescript/base",
+    "eslint:recommended",
+    "plugin:@cspell/recommended",
+    "plugin:@typescript-eslint/recommended",
+    // "plugin:@typescript-eslint/recommended-requiring-type-checking",
+    "plugin:import/recommended",
+    "plugin:import/typescript",
+    "plugin:prettier/recommended",
```

Comment:
We have prettier configured. I prefer disabling ESLint rules responsible for formatting and do the work in git commit hook rather than eslint autofix and red errors in IDE.

Reply:
It is formatted and fixed automatically thanks to the ESLint plugin for prettier 😎🤯🥳

Reply:
You can paste a malformed file and save and all ESLint rules takes precedence over Prettier rules... and prettier does not complain because the plugin :)

Reply:
No red errors in IDE. You can try and fix without worries

Reply:
I have tried and have errors while typing. It's a bad DX. Please use eslint config prettier instead.

Reply:
I'll need to fix :)